packages/server/.env.example

```diff
@@ -20,6 +20,7 @@ PORT=3000
 # FLOWISE_USERNAME=user
 # FLOWISE_PASSWORD=1234
 # FLOWISE_SECRETKEY_OVERWRITE=myencryptionkey
+# FLOWISE_FILE_SIZE_LIMIT=50mb
```

Comment:
do a search on other variable name like `FLOWISE_USERNAME`, there are other places we need to add this env var too